packages/unity-bootstrap-theme/src/scss/extends/_form-fields.scss

```diff
@@ -63,8 +63,7 @@ form.uds-form {
   textarea:focus,
```

Comment:
There is a side effect with targeting all input elements as it wraps text inputs too 
Left = code changes, Right = existing code
![image](https://github.com/user-attachments/assets/78b8516b-8f86-4bdc-b664-1af5ad8031e8)

Reply:
https://zeroheight.com/9f0b32a56/p/099ef8-text-input-field/b/07e355 style guide has example matching the right side

Reply:
Fixed, added special selector